types/react-base-utility/index.d.ts

```diff
@@ -0,0 +1,55 @@
+// Type definitions for react-base-utility 1.0
+// Project: https://github.com/AmeyKhoje/react-base-utility#readme
+// Definitions by: Amey Khoje <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/react-base-utility/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me

Reply:
Done